lib/radio_utils/image_message.py

```diff
@@ -0,0 +1,131 @@
+from .message import Message
+import os
+from .headers import IMAGE_START, IMAGE_MID, IMAGE_END
+
+class ImageMessage(Message):
+    """
+    encodes JPEG files into packets that can be transmitted over RF
+        - works for baseline DCT
+
+    You can find out if your JPEG image uses baseline DCT by looking at the start of frame
+    bytes. If they are FFC0, it is baseline otherwise it will be FFC2
+    """
+
+    headers = {
+        0xFF, 0xD8, 0xC0, 0xC2, 0xC4, 0xDA, 0xDB, 0xDD, 0xFE, 0xD9
+    }
+
+    # SIG = bytearray.fromhex("FF")
+    SIG = bytearray(1)
```

Comment:
what is SIG/SOS/EOI?

Reply:
These are specific bytes in the JPEG file format. All JPEG's start and end with the same bytes